DeepCSR/deepcsr/1.0/spec.yaml

```diff
@@ -1,41 +1,46 @@
-#### container info
 image:
+  docker: neuronets/deepcsr
   singularity: nobrainer-zoo_deepcsr.sif
-  docker: neuronets/nobrainer-zoo:deepcsr
-
-#### repository info
-repository:  
-    repo_url: "None"
-    committish: "None"
-    repo_download: False
-    repo_download_location: "None"
-    
-#### required fields for prediction
+repository:
+  repo_url: None
+  committish: None
+  repo_download: 'False'
+  repo_download_location: None
 inference:
-    prediction_script: "trained-models/DeepCSR/deepcsr/1.0/predict.py"
-    command: f"python {MODELS_PATH}/{model}/predict.py --conf_path {conf}"
-
-    #### input data characteristics
-    data_spec:
-      conf: {n_files: 1}
-
-#### model information and help
-model:
-    model_name: "deepcsr"
-    description: "3D deep learning framework for cortical surface reconstruction from MRI"
-    structure: "Meshnet"
-    training_mode: "supervised"
-    model_url: "https://github.com/neuroneural/DeepCSR-fork/"
-    Zoo_function: "predict"
-    example: "nobrainer-zoo predict -m DeepCSR/deepcsr/1.0 <path_to_input>"
-    note: "The output will be the stl files and will be saved in the config file's specified directory. For the input config file examples please see the /configs/predict.yaml file inside the model's repo."
-    input_file_type: "yaml"
-    model_details: ""
-    intended_use: ""
-    factors: ""
-    metrics: ""
-    eval_data: ""
-    training_data: ""
-    quant_analyses: ""
-    ethical_considerations: ""
-    caveats_recs: ""
\ No newline at end of file
+  prediction_script: trained-models/DeepCSR/deepcsr/1.0/predict.py
+  command: f"python {MODELS_PATH}/{model}/predict.py --conf_path {conf}"
+  data_spec:
+    infile:
+      n_files: 1
+    outfile:
+      n_files: 1
+training_data_info:
+  data_number:
+    total: 1
+    train: 1
+    evaluate: 1
+    test: 1
+  biological_sex:
+    male:
+    female:
+  age_histogram: '1'
```

Comment:
Why is this a string? I mean what is the difference between 1 and `1`?

Reply:
I just gave it as an input a random value. So that it passes the verification since I don't have in-depth information about the model at the moment.